samples/04_gis_analysts_data_scientists/land_cover_classification_using_sparse_training_data.ipynb

```diff
@@ -91,13 +91,10 @@
   },
```

Comment:
<p>need to revert this back</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/840/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>